gazebo_plugins/env-hooks/gazebo_plugins.sh.in

```diff
@@ -0,0 +1,3 @@
+# generated from gazebo_plugins/env-hooks/gazebo_plugins.sh.in
+
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:@GAZEBO_PLUGIN_PATH@
```

Comment:
I don't know if you care (not sure which platforms you're targeting), but on macOS the equivalent env var is `DYLD_LIBRARY_PATH` and on Windows it's just the `PATH` that needs to be set for `.dll`'s to be found.

Reply:
The `PATH` is updated already (so nothing new needs to be done there), but here's an example of us setting things conditionally with the library path:

https://github.com/ament/ament_package/blob/master/ament_package/template/environment_hook/library_path.sh

Reply:
Thanks, we shouldn't be breaking Mac or Windows, even though there's no CI running for them. Implemented your suggestion on 55303ac832b7ebd6a68243d5d53562e4752185fb and verified it doesn't break Ubuntu :grin: